.github/workflows/validate-pr.yml

```diff
@@ -75,3 +75,55 @@ jobs:
 
       - name: Jest test
         run: yarn test --passWithNoTests
+
+    lint:
```

Comment:
I _think_ this may need to be dedented one level. If I'm looking at this right, this looks like it falls under the `test` field, which we don't want.